src/streams/TopicItem.js

```diff
@@ -31,14 +31,7 @@ type Props = $ReadOnly<{|
 |}>;
 
 export default function TopicItem(props: Props) {
-  const {
-    name,
-    stream = '',
```

Comment:
🎉

The empty string at `stream` can't have been up to anything good. I'm curious what `TopicItem`'s caller in `UnreadCards` is trying to accomplish by passing the empty string, but that's probably for other work outside this PR.